<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GLinearCombinationWidget</class>
 <widget class="QWidget" name="GLinearCombinationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GLinearCombinationWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="pLayoutInput"/>
   </item>
   <item>
    <widget class="QTableView" name="pTableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>40</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pLayoutOutput"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
